### PR TITLE
Example for getFileFromPath function

### PR DESCRIPTION
**Pull request by notthemonkee**

### CODE REVIEW

File: data/en/getfilefrompath.json [hunk 1]
```diff
@@ -14,5 +14,14 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/getfilefrompath"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/getfilefrompath"}
 	},
-	"links": []
+	"links": [],
+	"examples" : [
+		{
+			"title":"Basic Example",
+			"description":"",
+			"code":"fileName = getFileFromPath(\"C:\\temp\\file.txt\"));",
```
Comment (pfreitag) [inline data/en/getfilefrompath.json]: for code you can simply use: `getFileFromPath(\"C:\\temp\\file.txt\"))`

File: data/en/getfilefrompath.json [hunk 2]
```diff
@@ -14,5 +14,14 @@
...
+			"result":"The fileName variable will contain the string file.txt",
```
Comment (pfreitag) [inline data/en/getfilefrompath.json]: Just set the result to the actual result, which is: `"file.txt"`

File: data/en/getfilefrompath.json [hunk 3]
```diff
@@ -14,5 +14,14 @@
...
+			"runnable":false
```
Comment (pfreitag) [inline data/en/getfilefrompath.json]: If the above changes are made this can be set runnable to `true`

Comment (notthemonkee) [reply data/en/getfilefrompath.json]: Got it, thanks @pfreitag I get how this works now.

**Comment by pfreitag** [comment]
Thanks for fixing... that change also allows the Travis-Ci task to validate that the result is correct as well when it runs its tests.

**Pull request merged**